package/MDAnalysis/analysis/encore/__init__.py

```diff
@@ -21,4 +21,14 @@
     'clustering'
 ]
 
-from .similarity import hes, ces, dres, ces_convergence, dres_convergence
+from .similarity import hes, ces, dres, \
+    ces_convergence, dres_convergence
+
+from .clustering.ClusterCollection import ClusterCollection, Cluster
+from .clustering.ClusteringMethod import *
+from .clustering.cluster import cluster
+from .dimensionality_reduction.DimensionalityReductionMethod import *
+from .dimensionality_reduction.reduce_dimensionality import (
+    reduce_dimensionality)
+from .confdistmatrix import get_distance_matrix
+from utils import merge_universes
```

Comment:
Please make this a relative import. (Add to the top
```python
from __future__ import absolute_import
```
to catch all of them.
Also add a trailing newline.